### PR TITLE
style: convert long comma lists after colons to OL

### PR DESCRIPTION
**Pull request by nschonni**
Did a quick search for long lines using `[^\n]{1000}` and noticed these few that seemed like long enough lists where an OL might be easier to read

### CODE REVIEW

File: files/en-us/web/guide/html/content_categories/index.md [hunk 1]
```diff
@@ -33,7 +33,90 @@ Elements belonging to this category are {{HTMLElement("base")}}, {{HTMLElement("
...
+- {{HTMLElement("sup")}}
+- {{SVGElement("svg")}}
+- {{HTMLElement("table")}}
+- {{HTMLElement("template")}}
+- {{HTMLElement("textarea")}}
+- {{HTMLElement("time")}}
+- {{HTMLElement("u")}}
+- {{HTMLElement("ul")}}
+- {{HTMLElement("var")}}
+- {{HTMLElement("video")}}, and {{HTMLElement("wbr")}}.
```
Comment (nschonni) [inline files/en-us/web/guide/html/content_categories/index.md:119]:
```suggestion
- {{HTMLElement("video")}}
- {{HTMLElement("wbr")}}
```

File: files/en-us/web/guide/html/content_categories/index.md [hunk 2]
```diff
@@ -62,7 +146,56 @@ Elements belonging to this category are {{HTMLElement("h1")}}, {{HTMLElement("h2
...
+- {{HTMLElement("strong")}}
+- {{HTMLElement("sub")}}
+- {{HTMLElement("sup")}}
+- {{SVGElement("svg")}}
+- {{HTMLElement("textarea")}}
+- {{HTMLElement("time")}}
+- {{HTMLElement("u")}}
+- {{HTMLElement("var")}}
+- {{HTMLElement("video")}}
+- {{HTMLElement("wbr")}} and plain text (not only consisting of white spaces characters).
```
Comment (nschonni) [inline files/en-us/web/guide/html/content_categories/index.md:198]: Maybe the text should shift to a separate bullet

File: files/en-us/web/svg/attribute/begin/index.md [hunk 3]
```diff
@@ -57,7 +57,41 @@ The `<begin-value-list>` is a semicolon-separated list of values. Each value can
...
+    - {{domxref("Element/compositionend_event", "compositionend")}}
+    - {{domxref("Window/load_event", "load")}}
+    - {{domxref("Window/unload_event", "unload")}}
+    - {{domxref("HTMLMediaElement/abort_event", "abort")}}
+    - {{domxref("Element/error_event", "error")}}
+    - {{domxref("HTMLInputElement/select_event", "select")}}
+    - {{domxref("Window/resize_event", "resize")}}
+    - {{domxref("Element/scroll_event", "scroll")}}
+    - {{domxref("SVGAnimationElement/beginEvent_event", "beginEvent")}}
+    - {{domxref("SVGAnimationElement/endEvent_event", "endEvent")}}, and {{domxref("SVGAnimationElement/repeatEvent_event", "repeatEvent")}}. An optional offset value as defined in `<offset-value>` can be appended.
```
Comment (nschonni) [inline files/en-us/web/svg/attribute/begin/index.md:93]:
```suggestion
    - {{domxref("SVGAnimationElement/endEvent_event", "endEvent")}}
    - {{domxref("SVGAnimationElement/repeatEvent_event", "repeatEvent")}}
    
    An optional offset value as defined in `<offset-value>` can be appended.
```

File: files/en-us/web/accessibility/aria/roles/index.md [hunk 4]
```diff
@@ -57,7 +57,22 @@ For most document structure roles, semantic HTML equivalent elements are availab
...
+These are included for completeness, but in most cases are rarely, if ever, useful:
```
Comment (nschonni) [inline files/en-us/web/accessibility/aria/roles/index.md:60]: I glued the opening and closing text of this block together

**Review by wbamberg — APPROVED** [review]
Very nice change!

**Pull request merged**